server/src/domain/ldap/group.rs

```diff
@@ -219,3 +233,48 @@ pub async fn get_groups_list<Backend: BackendHandler>(
         })
         .collect::<Vec<_>>())
 }
+
+#[cfg(test)]
+mod tests {
+    use ldap3_proto::LdapFilter;
+
+    use crate::domain::ldap::group::GroupRequestFilter;
+    use crate::domain::ldap::utils::{parse_distinguished_name, LdapInfo};
+
+    use super::convert_group_filter;
+
+    static BASE_DN_STR: &str = "dc=example,dc=com";
+
+    #[test]
+    fn test_convert_group_filter() {
```

Comment:
Oh, you put the tests there (which is where I told you to put the tests, actually...)

I think it'd better to add a test case to ldap_handler.rs where we test at a higher level the entire LDAP layer. Also all the setup is already done there.

Sorry for that!